src/main/java/uk/gov/hmcts/reform/sendletter/config/SchedulerConfiguration.java

```diff
@@ -2,40 +2,26 @@
 
 import net.javacrumbs.shedlock.core.LockProvider;
 import net.javacrumbs.shedlock.provider.jdbctemplate.JdbcTemplateLockProvider;
-import net.javacrumbs.shedlock.spring.ScheduledLockConfiguration;
-import net.javacrumbs.shedlock.spring.ScheduledLockConfigurationBuilder;
-import org.springframework.beans.factory.annotation.Value;
+import net.javacrumbs.shedlock.spring.annotation.EnableSchedulerLock;
 import org.springframework.boot.autoconfigure.AutoConfigureAfter;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.DependsOn;
-import org.springframework.scheduling.TaskScheduler;
+import org.springframework.scheduling.annotation.EnableScheduling;
 
-import java.time.Duration;
 import javax.sql.DataSource;
 
 @Configuration
 @AutoConfigureAfter(FlywayConfiguration.class)
 @DependsOn({"flyway", "flywayInitializer"})
 @ConditionalOnProperty(value = "scheduling.enabled", matchIfMissing = true)
+@EnableScheduling
```

Comment:
Do you need this ? It's already here https://github.com/hmcts/send-letter-service/blob/master/src/main/java/uk/gov/hmcts/reform/sendletter/Application.java#L10

Reply:
moved it to relevant place. comment on description:
> also moving `@EnableScheduling` to configuration instead where condition on property exists therefore enable will happen only if flag is on rather than per application scope